src/test/java/racingcar/model/CarNameTest.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.params.provider.ValueSource;
+import racingcar.message.ErrorMessage;
+
+class CarNameTest {
+
+    @DisplayName("자동차명은 1자에서 5자 이하이다.")
+    @ParameterizedTest
+    @ValueSource(strings = {"a", "ab", "abc", "abcd", "abcde"})
+    void valid_car_name(String carName) {
+        assertThatCode(() -> new Car(new CarName(carName)))
+                .doesNotThrowAnyException();
+    }
+
+    @DisplayName("자동차명이 null이면 exception을 발생시킨다.")
+    @Test
+    void car_name_with_blank() {
+        assertThatThrownBy(() -> new Car(new CarName("")))
```

Comment:
CarName 에 대한 테스트이니 `new Car` 는 제외하면 어떨까요?
```suggestion
        assertThatThrownBy(() -> new CarName(""))
```